build.gradle

```diff
@@ -77,8 +79,8 @@ subprojects {
     dependencies {
         compile "org.slf4j:slf4j-api:$slf4jVersion"
         compile "org.slf4j:jul-to-slf4j:$slf4jVersion"
-        compile "ch.qos.logback:logback-classic:$logbackVersion"
-        compile "ch.qos.logback:logback-core:$logbackVersion"
+        compile "ch.qos.logback:logback-classic:$logbackVersion", optional
+        compile "ch.qos.logback:logback-core:$logbackVersion", optional
```

Comment:
Why is logback optional?

Reply:
I had to double check this as it surprised me when I saw it was set as optional in 2.5.1, so the Logback dependency isn't forced on the 3rd party using jbake-core. The 3rd party is free to use any framework they like.